### ISSUE TITLE
Combine sigstore FAQs into one FAQ /check links and help page

### ISSUE THREAD

**Issue by jonvnadelberg** [sigstore/docs#176]
**Description**

There are several FAQs under https://docs.sigstore.dev/  They should be combined into one FAQ with headers for different sections and be placed under the help header.

All links in sigstore docs should be checked to see if they currently work and go to a reasonable location.

Help page maintainers should be checked to see if they are still accurate.

**Comment by jonvnadelberg** [comment]
@haydentherapper @olivekl @ltagliaferri 

Discussion of FAQ on sigstore-website

I'm trying to coordinate the FAQs between sigstore-website and sigstore-docs.   So far I think we need to keep an FAQ for the sigstore-website, but move some of the questions over to an FAQ on the docs side.   So I think the following would be a good set to keep on the website, and move the rest over to docs:

Keeping on website:
"question": "What can I sign and store?",
 "question": "Are there any privacy issues?",
"question": "What's the current status?",
"question": "How easy is it to use?",
"question": "How can people get involved?",

Moving to docs:
"question": "What security checks do you use internally?",
"question": "Is there live support?",
"question": "Is the transparency log monitored?"
"question": "Can I run my own transparency log monitor?",
"question": "Why not blockchain?",
"question": "How does Sigstore integrate in-toto?",
"question": "How does Sigstore use / interact with TUF?",


I'm planning on moving all the FAQs together and put headers on them so that they are more easily found, as they currently are all over the place.

Question regarding sigstore-website, does anyone keep this updated?  It appears to not have been touched in a year and a half.

Anyway, any comments or suggestions regarding this part of the task would be helpful.

Thanks!

**Comment by olivekl** [comment]
Thanks for this! A few minor points on the questions:

"Is there live support?" -- I think this makes sense to leave on the website
"Are there any privacy issues?" -- potentially drop
"Why not blockchain?" -- potentially drop

Happy to hear thoughts from @haydentherapper @ltagliaferri on these suggestions!

**Comment by jonvnadelberg** [comment]
By dropping you mean getting rid of completely?

**Comment by ltagliaferri** [comment]
I would rephrase the TUF question to be something like this:

> "What is the relationship between Sigstore and TUF?"

Just because it sounds like you may be addressing both the "use" and "interact with" portions of the original question.

The [sigstore-website repo ](https://github.com/sigstore/sigstore-website) was last updated two weeks ago. If you want to open a PR there, I can review and merge. It is not a big priority across contributors because it is more of a "showcase" site I believe, and the blog is contained in [sigstore-blog](https://github.com/sigstore/sigstore-blog).

**Comment by jonvnadelberg** [comment]
On another subject dealing with this issue, I want to combine all the Sigstore-docs FAQs along with the general FAQs into one FAQ under the **Help** section.   Each of the current FAQs (one for cosign, fulcio, rekor, and general from sigsore-website that gets pulled out as discussed above) would go into this new FAQ.   The FAQ will have headers for each of these topics (again: cosign, fulcio,rekor,general) and have the current questions answered there.

Anyone have thoughts on this plan?
